lib/api.js

```diff
@@ -227,8 +228,6 @@ function transformOptions(options, encoding) {
   };
 
   if (options.contentType !== undefined) {
```

Comment:
I think we can just remove this guard now, and ignore options.contentType for the purposes of this function. I'll try it.